gpkit/tests/t_model.py

```diff
@@ -311,6 +311,16 @@ def test_sp_initial_guess_sub(self):
                       "and variables failed")
         self.assertAlmostEqual(sol["cost"], 1, self.ndig)
 
+    def test_while_loop_exit_condition(self):
+        with EnableSignomials():
+            x = Variable('x')
+            z = Variable('z')
+            J = 0.01*(x - 1)**2
+            sp = Model(z, [z >= J])
+            sol = sp.localsolve(verbosity=0, iteration_limit=50)
```

Comment:
do you mean just `sp.gps`? I get an AttributeError with `program` and `len(sp.gps)` gives me what I want (`22`).